L/libjlnode/build_tarballs.jl

```diff
@@ -3,11 +3,11 @@
 using BinaryBuilder, Pkg
 
 name = "libjlnode"
-version = v"16.14.0"
+version = v"16.0.0"
```

Comment:
I think this is going to build a new JLL that has an _older_ version number than the last one.  Is that what you want?

Reply:
The last registered one is 14.17.3 so 16.0.0 is not older :) It is want I intended to do

Reply:
Ah, I was confused because I was looking at only one commit, not the whole thing. :)